pjsip-apps/src/swig/java/android/app-kotlin/src/main/java/org/pjsip/pjsua2/app_kotlin/MainActivity.kt

```diff
@@ -20,10 +20,15 @@ import java.lang.ref.WeakReference
 /* Configs */
 
 // Account ID
-const val ACC_ID_URI    = "sip:localhost"
+const val ACC_DOMAIN = "pjsip.org"
+const val ACC_USER   = "101"

```

Comment:
usually we use test account here ("test"), also for the destination

Reply:
I used three accounts for the doc, hence I thought using 102-103 has better symmetry to it.